bundles/org.openhab.binding.wundergroundupdatereceiver/README.md

```diff
@@ -0,0 +1,88 @@
+# WundergroundUpdateReceiver Binding
+
+Many personal weather stations are only capable of submitting measurements to the wunderground.com update site.
+
+The request is in itself simple to parse, so by redirecting it to your openhab server you can intercept the values and use them to
+control items in your home. Fx. use measured wind-speed to close an awning or turn on the sprinkler system after some time without rain.
+This binding allows you to mix and match products from various manufacturers that otherwise have a closed system.
+
+It can also be used to submit the same measurements to multiple weather services in rules.
+
+## Supported Things
+
+Any weather station that sends weather measurement updates to the wunderground.com update URLs.
+It is easiest to use with stations that have a configurable target address, but can be made to
+work with any internet-connected weather station, that gets it's dns server via DHCP. 
+
+## Discovery
+
+In the initial version ther is no discovery, but it is planned for a later release. You need to manually add a thing and configure the station id.
+
+## Thing Configuration
+
+The only configurable value is the station id, which should match the one configured on the weather station. If you don't plan on submitting
+measurements to wunderground.com it can be any unique non-empty string value.
+
+## Channels
+
+Each measurement type the wunderground.com update service accepts has a channel. Additionally there is a receipt timestamp and a trigger channel.
+
+State channels:
+
+| channel  | type   | description                  |
+|----------|--------|------------------------------|
+| control  | Switch | This is the control channel  |
+| last-received-datetime | DateTime | The date and time of the last update. |
+| datetime-utc | String | The date and time of the last update in UTC as submitted by the weather station. This can be 'now'. |
+| wind-speed | Number:Speed | Current wind speed, using software specific time period. |
+| wind-speed-avg-2min | Number:Speed | 2 minute average wind speed. |
+| wind-gust-speed-10min | Number:Speed | 10 minute average gust speed. |
```

Comment:
my weather station is not reporting those average values therefore they remain `null`.
Before reading the code I was expecting, that they are calcualated by the binding.
You may want to leave a comment in the description...

Reply:
Yes when I get some time, I'll implement discovery, which will generate the channels the given weather station submits values for, but for the time being I calculate these in the rules used to effect other things, i.e. only fiddling with awnings if the avg gust speed has been high for a minute.